dockstore-webservice/src/main/java/io/dockstore/webservice/core/metrics/ExecutionsRequestBody.java

```diff
@@ -40,7 +41,8 @@ public class ExecutionsRequestBody {
 
     @NotNull
     @Valid
-    @Schema(description = "List of aggregated executions to submit")
+    @ArraySchema(arraySchema = @Schema(description = "List of aggregated executions to submit", deprecated = true))
```

Comment:
Had to change this from `@Schema` to `@ArraySchema` because adding `deprecated = true` to `@Schema` also deprecated the `Metrics` schema, which we don't want

Reply:
Interesting that openapi didn't change much.

Reply:
Could also do `@Hidden` till we get a request